training/src/anemoi/training/schemas/base_schema.py

```diff
@@ -89,6 +89,24 @@ def check_log_paths_available_for_loggers(self) -> BaseSchema:
             raise PydanticCustomError("logger_path_missing", msg)  # noqa: EM101
         return self
 
+    @model_validator(mode="after")
+    def check_bounding_not_used_with_data_extractor_zero(self) -> BaseSchema:
+        """Check that bounding is not used with zero data extractor."""
+        if (
+            self.model.decoder.target_ == "anemoi.models.layers.mapper.GraphTransformerBackwardMapper"
```

Comment:
If this symbol is ever moved or renamed and we forget to update this string comparison, this check will silently pass all the time. How should we handle that? Ideally we should check if this symbol actually exists, but we also don't want to import it if it's not being used. 🤔

Don't let this block the PR, it looks good to merge. Just something to think about.